presto-hive/src/main/java/com/facebook/presto/hive/HiveSplit.java

```diff
@@ -60,13 +62,14 @@ public HiveSplit(
             @JsonProperty("start") long start,
             @JsonProperty("length") long length,
             @JsonProperty("fileSize") long fileSize,
-            @JsonProperty("schema") Properties schema,
+            @JsonProperty("storage") Storage storage,
```

Comment:
Curious: why we need to pass `Storage` now?  Can `serdeParameter` be a huge map?

Reply:
Also what's suppose to be stored in `schema` when it's a `Properties`

Reply:
> Curious: why we need to pass Storage now? Can serdeParameter be a huge map?

This object contains partition specific storage information. Unfortunately if partition has some custom storage parameter - we should pass it, as those are needed to initialize the Input reader / serde.

Reply:
> Also what's suppose to be stored in schema when it's a Properties

Yeah, the `Properties schema` used to contain them before.